options.mk

```diff
@@ -15,12 +15,13 @@ ifeq ($(SIGN),ECC256)
     ./lib/wolfssl/wolfcrypt/src/ecc.o \
     ./lib/wolfssl/wolfcrypt/src/memory.o \
     ./lib/wolfssl/wolfcrypt/src/wc_port.o \
+    ./lib/wolfssl/wolfcrypt/src/wolfmath.o \
     ./lib/wolfssl/wolfcrypt/src/hash.o
-  CFLAGS+=-DWOLFBOOT_SIGN_ECC256 -DXMALLOC_USER
+  CFLAGS+=-D"WOLFBOOT_SIGN_ECC256"
```

Comment:
Did you mean to remove the XMALLOC_USER?

Reply:
No, I've generalized this so it can be overridden and done that for the R5.